docs/configurations.md

```diff
@@ -186,6 +186,10 @@ The following are additional configurations available for ScalarDB:
 | `scalar.db.default_namespace_name`                               | The given namespace name will be used by operations that do not already specify a namespace.                                                                                                |                      |
 | `scalar.db.system_namespace_name`                                | The given namespace name will be used by ScalarDB internally.                                                                                                                               | `scalardb`           |
 
+## Use placeholders
+
+You can use placeholders in the values, and they are replaced with environment variables (${env:<ENVIRONMENT_VARIABLE_NAME>}) or system properties (${sys:<SYSTEM_PROPERTY_NAME>}). You can also specify default values in placeholders like ${sys:<SYSTEM_PROPERTY_NAME>:-<DEFAULT_VALUE>}.
```

Comment:
> environment variables (${env:<ENVIRONMENT_VARIABLE_NAME>})

How about syntax-highlighting it as follows?

```md
environment variables (`${env:<ENVIRONMENT_VARIABLE_NAME>}`)
```

Reply:
Ah, I see. Looks good!
I will apply it. Thank you for your suggestion!